src/main/scala/diplomaticobjectmodel/logicaltree/LogicalTrees.scala

```diff
@@ -0,0 +1,89 @@
+// See LICENSE.SiFive for license details.
+
+package freechips.rocketchip.diplomaticobjectmodel.logicaltree
+
+import freechips.rocketchip.devices.debug._
+import freechips.rocketchip.devices.tilelink.PLICParams
+import freechips.rocketchip.diplomacy._
+import freechips.rocketchip.diplomaticobjectmodel.DiplomaticObjectModelAddressing
+import freechips.rocketchip.diplomaticobjectmodel.model.{OMComponent, _}
+import freechips.rocketchip.rocket.{DCacheParams, ICacheParams, ScratchpadSlavePort}
+
+class CLINTLogicalTreeNode(device: SimpleDevice, f: () => OMRegisterMap) extends LogicalTreeNode {
```

Comment:
Did you try making this: `class CLINTLogicalTreeNode(device: SimpleDevice, f: => OMRegisterMap)` and then getting rid of the extra `getOMRegMap` accessor?

Reply:
fixed. lets go over this on Monday for 30 seconds...